cmd/containerd-stargz-grpc/main.go

```diff
@@ -192,3 +198,17 @@ func hostsFromConfig(cfg ResolverConfig, keychain authn.Keychain) remote.Registr
 		return
 	}
 }
+
+func sources(ps ...source.GetSources) source.GetSources {
+	return func(labels map[string]string) (source []source.Source, _ error) {
+		allErr := fmt.Errorf("cannot parse labels")
+		for _, p := range ps {
+			src, err := p(labels)
+			if err == nil {
+				return src, nil
+			}
+			allErr = errors.Wrapf(allErr, "%v;", err)
```

Comment:
Can we use https://github.com/hashicorp/go-multierror ?

Reply:
Thanks. Fixed to use go-multierrror.